example/_src/page-template.html

```diff
@@ -28,47 +28,70 @@
     @google-universal-analytics["UA-xxxxx"]
   </head>
   <body>
+
     <!-- A standard Twitter Bootstrap nav bar -->
-    <header class="navbar navbar-default navbar-inverse"
-            role="banner">
+    <nav class="navbar navbar-expand-md navbar-dark bg-dark">
       <div class="container">
-        <div class="navbar-header">
-          <button type="button"
-                  class="navbar-toggle"
-                  data-toggle="collapse"
-                  data-target=".our-nav-collapse">
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-          </button>
-          <a href="@|uri-prefix|/index.html" class="navbar-brand">My Blog Brand</a>
-        </div>
-        <div class="collapse navbar-collapse our-nav-collapse"
-             role="navigation">
-          <ul class="nav navbar-nav">
+
+      <a href="@|uri-prefix|/index.html" class="navbar-brand">My Blog Brand</a>
+
+      <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse"
+              data-target="#navbar_collapse" aria-controls="navbar_collapse"
+              aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <div class="collapse navbar-collapse" id="navbar_collapse">
+          <ul class="navbar-nav mr-auto">
             @;{ We can define Racket functions to "DRY". Here's one to
                 emit HMTL for a nav item, adding class="active" iff
                 this page's @|uri-path| is the same as the nav link. }
+
             @(define (ni uri label [a-attribs ""])
               @list{
                 <li@(when (string-ci=? uri uri-path) " class=\"active\"")>
-                  <a href="@|uri|"@|a-attribs|>@|label|</a>
+                  <a class="nav-link" href="@|uri|"@|a-attribs|>@|label|</a>
                 </li> })
-            <li class="dropdown">
-              <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-                Tags <b class="caret"></b></a>
-              <ul class="dropdown-menu">
-                <li><a href="@|uri-prefix|/index.html">All Posts</a></li>
-                @|tags-list-items|
-              </ul>
+
+
+            @;{ Here we craft the dropdown links from a list of pairs.
+                The pairs are in the format '((name . url) ...) }
+
+            @(define (craft-tag-links listof-pairs)
+              @(map (lambda (pair)
+                @list{
+                <a class="dropdown-item" href="@(cdr pair)">
+                  @(car pair)
+                </a>})
+              listof-pairs)
+            )
```

Comment:
This dangling closing paren jumps out at me. (But I wouldn't not merge the PR just because of this. :smile:)

Reply:
Yeah, I think I'm gonna change that section a bit.